scan/src/components/msgs/Msg.re

```diff
@@ -201,7 +201,7 @@ let make = (~msg: TxSub.Msg.t, ~width: int) => {
     </div>
   | RequestMsg({id, oracleScriptID, oracleScriptName, sender}) =>
     <div className={Styles.rowWithWidth(width)}>
-      <div className={Styles.withWidth(130)}> <AddressRender address=sender /> </div>
+      <div className={Styles.withWidth(120)}> <AddressRender address=sender /> </div>
```

Comment:
I think we should adjust the width on all msgs which have an address.

![image](https://user-images.githubusercontent.com/17017699/90418335-3503ed00-e0df-11ea-8f15-bba9f5808b0c.png)